src/IECoreScene/MeshPrimitive.cpp

```diff
@@ -393,16 +394,19 @@ MeshPrimitivePtr MeshPrimitive::createPlane( const Box2f &b, const Imath::V2i &d
 			vIds.push_back( v1 );
 			vIds.push_back( v2 );
 			vIds.push_back( v3 );
-
-			uvs.emplace_back( j * uStep, i * vStep );
-			uvs.emplace_back( (j+1) * uStep, i * vStep );
-			uvs.emplace_back( (j+1) * uStep, (i+1) * vStep );
-			uvs.emplace_back( j * uStep, (i+1) * vStep );
 		}
 	}
 
 	MeshPrimitivePtr result = new MeshPrimitive( verticesPerFace, vertexIds, "linear", pData );
-	result->variables["uv"] = PrimitiveVariable( PrimitiveVariable::FaceVarying, uvData );
+	// We could use Vertex interpolation here, since that's logicaly the same
+	// as FaceVarying interpolation using `vertexIds` as the indices. We don't
+	// do that for a few reasons :
+	//
+	//   - Most DCCs represent UVs using explicit indices anyway
+	//   - Some of our code is currently incompatible with Vertex UVs
+	//   - This gives us extra test coverage for indexed UVs "for free",
+	//     since several tests use `createPlane()`.
+	result->variables["uv"] = PrimitiveVariable( PrimitiveVariable::FaceVarying, uvData, vertexIds );
```

Comment:
@donboie, I remember you said that you thought these should be Vertex. I don't disagree in theory, but I've commented some practical reasons why I think sticking with FaceVarying is best right now. Also note that they were already FaceVarying anyway, so this change is an improvement on the status quo : it reduces the amount of data and gives us proper connectivity information that was missing before.